### DISCUSSION TITLE
Issue installing

### DISCUSSION THREAD

**Discussion by ashima-del** [Sunbird-Knowlg/Community#214] [Q&A]
I am trying  to build sunbird - knowlg on local machine and facing the following issue:

It is expecting some variables in a pom file and failing:

CLOUD_STORE_GROUP_ID
CLOUD_STORE_ARTIFACT_ID
CLOUD_STORE_VERSION

What are these to be set to?

NIELIT@Nielit1:~/knowledge-platform$ mvn clean install -DskipTests
[INFO] Scanning for projects...
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[ERROR] 'dependencies.dependency.artifactId' for ${CLOUD_STORE_GROUP_ID}:${CLOUD_STORE_ARTIFACT_ID}:jar with value '${CLOUD_STORE_ARTIFACT_ID}' does not match a valid id pattern. @ line 32, column 25
[ERROR] 'dependencies.dependency.groupId' for ${CLOUD_STORE_GROUP_ID}:${CLOUD_STORE_ARTIFACT_ID}:jar with value '${CLOUD_STORE_GROUP_ID}' does not match a valid id pattern. @ line 31, column 22
[ERROR] 'dependencies.dependency.version' for ${CLOUD_STORE_GROUP_ID}:${CLOUD_STORE_ARTIFACT_ID}:jar must be a valid version but is '${CLOUD_STORE_VERSION}'. @ line 33, column 22
 @ 
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]   
[ERROR]   The project org.sunbird:mimetype-manager:1.0-SNAPSHOT (/home/NIELIT/knowledge-platform/platform-modules/mimetype-manager/pom.xml) has 3 errors
[ERROR]     'dependencies.dependency.artifactId' for ${CLOUD_STORE_GROUP_ID}:${CLOUD_STORE_ARTIFACT_ID}:jar with value '${CLOUD_STORE_ARTIFACT_ID}' does not match a valid id pattern. @ line 32, column 25
[ERROR]     'dependencies.dependency.groupId' for ${CLOUD_STORE_GROUP_ID}:${CLOUD_STORE_ARTIFACT_ID}:jar with value '${CLOUD_STORE_GROUP_ID}' does not match a valid id pattern. @ line 31, column 22
[ERROR]     'dependencies.dependency.version' for ${CLOUD_STORE_GROUP_ID}:${CLOUD_STORE_ARTIFACT_ID}:jar must be a valid version but is '${CLOUD_STORE_VERSION}'. @ line 33, column 22
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException

**Comment by aimansharief** [comment]
@ashima-del can you please refer the below link and follow the steps accordingly.
https://github.com/Sunbird-Knowlg/knowledge-platform?tab=readme-ov-file#running-content-service

**Reply by pallakartheekreddy** [reply]
@ashima-del Please let us know if your query is resolved
